src/Symfony/Bundle/FrameworkBundle/Tests/Functional/NotificationTest.php

```diff
@@ -11,11 +11,11 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Tests\Functional;
 
+use PHPUnit\Framework\Attributes\RequiresFunction;
+
 final class NotificationTest extends AbstractWebTestCase
 {
-    /**
-     * @requires function \Symfony\Bundle\MercureBundle\MercureBundle::build
-     */
+    #[RequiresFunction('Symfony\Bundle\MercureBundle\MercureBundle::build')]
```

Comment:
Should `RequiresMethod` be used instead:


```suggestion
    #[RequiresMethod(MercureBundle::class, 'build')]
```